pkg/trtl/honu.go

```diff
@@ -11,15 +14,53 @@ import (
 // A HonuService implements the RPCs for interacting with a Honu database.
 type HonuService struct {
 	pb.UnimplementedTrtlServer
+	parent *Server
+	db     *honu.DB
 }
 
-func NewHonuService() *HonuService {
-	return &HonuService{}
+func NewHonuService(s *Server) (*HonuService, error) {
+	return &HonuService{parent: s, db: s.db}, nil
 }
 
 // Get is a unary request to retrieve a value for a key.
 func (h *HonuService) Get(ctx context.Context, in *pb.GetRequest) (out *pb.GetReply, err error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+	if in.Options.ReturnMeta {
```

Comment:
do we need to check if `in.Options` is nil in case of a `nil` panic? It could be that gRPC always gives back an empty struct here, but it would be good to check.

Reply:
Good call.